d2bs/kolbot/libs/common/Misc.js

```diff
@@ -1,2414 +1,2417 @@
 /**
-*	@filename	Misc.js
-*	@author		kolton
-*	@desc		misc library containing Skill, Misc and Sort classes
-*/
+ *    @filename    Misc.js
```

Comment:
This file needs to be resubmitted without the spacing differences.

Reply:
This is exactly the resubmitted version after my other commit that ruined the format. So, by reverting it to the old version, git thinks that it is again a completely new file since its an exact copy of the current Misc.js